src/renderer/helpers/history.js

```diff
@@ -0,0 +1,4 @@
+export const SORT_BY_VALUES = {
+  DateAddedNewest: 'newestFirst',
+  DateAddedOldest: 'oldestFirst',
+}
```

Comment:
Please move this constant into the `src/constants.js` file and add `HISTORY` to its name (not worth creating a new file for a single constant, especially when we already have a file for constants that are used in multiple places).